core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -2436,6 +2437,32 @@ Scan *RelExpr::getLeftmostScanNode() const
   return (Scan *)result;
 }
 
+
+
+Join * RelExpr::getLeftJoinChild() const
+{
+  RelExpr *result = (RelExpr *)this;
+  
+  while(result)
+    {
+      if (result->getOperatorType() == REL_LEFT_JOIN) break;
+      result = result->child(0);
```

Comment:
This is cloned from the existing method getLeftMostScanNode(). They can atmost return NULL which is ok.